src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -250,21 +247,25 @@ def get(self, request: Request, organization, tag_name) -> Response:
         if not projects:
             raise InvalidParams("You must supply at least one project to see the tag values")
 
-        start, end = get_date_range_from_params(request.GET)
-
         try:
-            tag_values = get_tag_values(
-                projects=projects,
-                tag_name=tag_name,
-                metric_names=metric_names,
-                use_case_id=get_use_case_id(request),
-                start=start,
-                end=end,
-            )
+            mris = convert_metric_names_to_mris(metric_names)
```

Comment:
I would drop the support for multiple metric names and restrict the API to a single one. If we make sequential queries for each metric name, we might timeout the request since it takes too long.

Reply:
Added a check and raised an exception if there is more than one metric name in the list - keeping the logic name the same so we can go back if we need it shortly; otherwise I'd then simplify once we know that there are no adverse effects from this change.